packages/tools/pluggable-widgets-tools/configs/rollup.config.js

```diff
@@ -241,25 +245,31 @@ export default async args => {
                   })
                 : null,
             image(),
-            production ? terser() : null,
+            production
+                ? terser({
```

Comment:
This change can be removed, as we fixed it here https://github.com/mendix/widgets-resources/pull/929/files